docs/checks.md

```diff
@@ -303,5 +303,57 @@ In line 3, column 14: PROTO and MODULE inputs should be in seperate groups. [imp
 In line 3, column 21: "random" is already defined. [imports]
 ```
 
+## expression-parantheses
+Use parathenthesis to highlight the order when there are multiple Boolean operators of different types.
 
+**Example**
+```sql
+-- Here is ok because they are the same type.
+SELECT D WHERE a AND b AND c;
+
+-- Here is an invalid usage of operators.
+SELECT D WHERE a AND b OR c; -- line 5
+
+-- It can be fixed by adding paratheses.
+SELECT D WHERE a AND (b OR c);
+```
+
+**Linter Output**
+```
+In line 5, column 16: Use parantheses between consequtive AND and OR statements. [expression-parantheses]
```

Comment:
Update the error message, since you've updated it in the code

Reply:
fixed